code/game/objects/effects/effect_system/effects_explosion.dm

```diff
@@ -61,3 +61,28 @@
 /datum/effect_system/explosion/smoke/start()
 	..()
 	addtimer(CALLBACK(src, PROC_REF(create_smoke)), 5)
+
+/obj/effect/explosion/delamination
+	name = "delamination"
+	icon_state = "sm_shatter"
+	plane = ABOVE_LIGHTING_PLANE
```

Comment:
Remove TILE_BOUND from the vis flags. This will cause it to do a more expensive visability calculation, however it will allow you to see the explosion even if you can't see the source of the explosion effect. Its more expensive, but should be fine since there is only 1 and it lasts for just 1.8 seconds.

Reply:
Done